### PR TITLE
news v2.0

### PR DESCRIPTION
**Pull request by Tomek1592**
- Chenge password design,
- news, newsDetails redesign,
- news publish date asc order,
- news details next news clickable + asc order,
- sidebar company info design cosmetic changes

### CODE REVIEW

File: src/components/News.vue [hunk 1]
```diff
@@ -5,26 +5,40 @@
...
         <h2 v-for="error of errors">{{ error.message }}</h2>
       </div>
-        <h5 class="news-header">Aktualności:</h5>
-        <hr>
-      <div v-for="item of news" :key="item.id" class="news-item clearfix">
-        <div class="news-date">
-          {{ $t('news.added') }} {{ convertTimeStamp }}
+      <h5 class="news-header">{{ $t('news.news') }}</h5>
+      <hr>
+      <div v-for="item of news" :key="item.id" class="row panel-item clearfix" v-show="item.published > Date.now() - 13046400000">
```
Comment (shaz950) [inline src/components/News.vue]: Consider putting this comparision `
> item.published > Date.now() - 13046400000"
in method that will take as a argument item.published and will return true or false
`

File: src/components/News.vue [hunk 2]
```diff
@@ -5,26 +5,40 @@
...
+            </div>
+          </div>
+          <div class="row">
+            <div class="col panel-content">
+              <p>{{ shortenContent(item.contents) }}</p>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col">
+              <router-link :to="{ name: 'news', params: { id: item.id }, query: { type: 'RELEASED', shortened: false } }" class="btn news-button">
```
Comment (bwielgosz) [inline src/components/News.vue]: Unecessary query object . /news/id API require only id

File: src/components/News.vue [hunk 3]
```diff
@@ -37,42 +51,67 @@
...
+        return date.format(this.$t('common.dateFormat'))
+      },
+      isNewsExpired (itemPublished) {
+        if (itemPublished > Date.now() - 13046400000) {
+          return true
+        } else {
+          return false
+        }
+      },
+      sortNews (key, order = 'asc') {
```
Comment (bwielgosz) [inline src/components/News.vue]: /news endpoint responses with ordered news(if isOrdered parameter set to true), but if you want sort by other dates(for example last modified) why use just an array sort method with compare function?

File: src/components/CompanyInfo.vue [hunk 4]
```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="sidebar-panel">
+    <p class="sidebar-panel-header">{{ $t('news.rightPanel.companyName') }}</p>
```
Comment (bwielgosz) [inline src/components/CompanyInfo.vue]: wrong i18n namespace

File: src/components/News.vue [hunk 5]
```diff
@@ -1,33 +1,43 @@
...
-          <h5 class="news-header font-weight-bold">{{ $t('news.news') }}:</h5>
-          <hr>
-          <div v-for="item of news" :key="item.id" class="news-item clearfix">
-            <p class="news-date">
-              {{ $t('news.added') }} {{ convertTimeStamp(item.published) }}
-            </p>
-            <h2 class="news-title">
-              <router-link :to="{ name: 'news', params: { id: item.id } }">
-                {{ item.title }}
+  <div class="container row wrapper">
```
Comment (bwielgosz) [inline src/components/News.vue]: row should be inside container class

File: src/components/News.vue [hunk 6]
```diff
@@ -1,33 +1,43 @@
...
+            </div>
+          </div>
+          <div class="row">
+            <div class="col">
+              <p class="float-left">{{ item.contents }}</p>
+            </div>
+          </div>
+          <div class="row mt-5">
+            <div class="col">
+              <router-link :to="{ name: 'news', params: { id: item.id }, query: { type: 'RELEASED', isShortened: false, isOrdered: true } }" class="btn news-button">
```
Comment (bwielgosz) [inline src/components/News.vue]: redundant query object

Comment (Tomek1592) [inline src/components/News.vue]: Can you explain?

Comment (bwielgosz) [inline src/components/News.vue]: GET /news/id API require only one parameter: id. You don't have to pass in link other parameters like type, isShortened or isOrdered

Comment (Tomek1592) [inline src/components/News.vue]: :to="{ name: 'news', params: { id: item.id } }"

You mean this?

Comment (bwielgosz) [inline src/components/News.vue]: Yes, indeed :)

File: src/components/News.vue [hunk 7]
```diff
@@ -1,33 +1,43 @@
...
+    <div class="col-12 col-lg-9">
+      <loader v-if="loading"></loader>
+      <div class="errors" v-if="errors.length">
+        <h2 v-for="error of errors">{{ error.message }}</h2>
+      </div>
+      <h5 class="news-header">{{ $t('news.news') }}</h5>
+      <hr>
+      <div v-for="item of news" :key="item.id" class="row panel-item" v-show="isNewsExpired(item.published)">
+        <div class="col-3 hidden-md-down">
+          <img :src="item.imageUrl" alt="" class="panel-img float-left">
```
Comment (shaz950) [inline src/components/News.vue]: This src wont work, now path is /storage/id*, it points to nothing, u need to concat ApiUrl before the imageUrl

**Pull request merged**